opensrp-app/src/main/java/org/smartregister/sync/helper/SyncSettingsServiceHelper.java

```diff
@@ -20,6 +20,8 @@
 
 import timber.log.Timber;
 
+import static org.smartregister.util.JsonFormUtils.isBlankJsonArray;
```

Comment:
Avoid static imports whenever possible.

https://docs.oracle.com/javase/7/docs/technotes/guides/language/static-import.html

Reply:
Cool, yea I think the point is valid. I'd argue this is sparingly used and the readability is not affected?